homeassistant/components/onewire/binary_sensor.py

```diff
@@ -164,7 +172,7 @@ class OneWireBinarySensorEntity(OneWireEntity, BinarySensorEntity):
 
     @property
     def is_on(self) -> bool | None:
-        """Return true if sensor is on."""
+        """Return true if sensor is on (present)."""
```

Comment:
There is no need to change this.
```suggestion
        """Return true if sensor is on."""
```